lib/msal-browser/src/broker/msg/resp/BrokerHandshakeResponse.ts

```diff
@@ -35,8 +36,11 @@ export class BrokerHandshakeResponse extends BrokerMessage {
             message.data.messageType === BrokerMessageType.HANDSHAKE_RESPONSE &&
             message.data.version) {
             // TODO, verify version compatibility
-            if (trustedBrokerDomains.indexOf(message.origin) < 0) {
-                // TODO make this a browser Error
+            const matchedDomains = trustedBrokerDomains.filter((domain: string) => {
+                return StringUtils.matchPattern(domain, message.origin);
```

Comment:
There's a `getDomainFromUrl` function on `UrlString` which may safer than just checking if one string is in another string

Reply:
I'm using `StringUtils.matchPattern()` so that we can support wildcard patterns in the config (i.e. `https://*.contoso.com`).

Reply:
Gotcha, thanks!